tests/no-internal.js

```diff
@@ -61,14 +61,7 @@ ruleTester.run(
         `,
         errors: [
           { message: 'function "internal" is internal.' },
-          // FIXME: there is separate work for removing duplicate violations
           { message: 'class "Internal" is internal.' },
```

Comment:
is it just me or does this remove too many instances?  I see 3 remaining but shouldn't there be 4? (`internal()`, `new Internal()`, `new Internal().publicMethod()`, `new Public().internalMethod()`)?

Reply:
Yes, I misinterpreted the test file. Will try to understand the no-internal rule better and debug

Reply:
looks correct to me now, thanks!